website/src/pages/roadmap.md

```diff
@@ -4,69 +4,55 @@ last_modified_at: 2019-12-30T15:59:57-04:00
-|Non-keyed tables with updates and deletes |0.11|[HUDI-2968](https://issues.apache.org/jira/browse/HUDI-2968)|
-|Trino Connector for Hudi, with read/query support  |0.12|[HUDI-2687](https://issues.apache.org/jira/browse/HUDI-2687), [RFC-38](https://github.com/apache/hudi/pull/3964)|
-|Spark Datasource V2|0.12|[HUDI-1297](https://issues.apache.org/jira/browse/HUDI-1297) ,[HUDI-2531](https://issues.apache.org/jira/browse/HUDI-2531)|
-|Complete ORC Support across query engines |0.12|[HUDI-57](https://issues.apache.org/jira/browse/HUDI-57)|
-|Presto Connector for Hudi, with read/query support |0.12|[PRESTO-17006](https://github.com/prestodb/presto/issues/17006)|
-|Multi-Modal indexing full integration across Presto/Trino/Spark queries |0.12 -> 1.0.0|[HUDI-1822](https://issues.apache.org/jira/browse/HUDI-1822)|
-|Materialized Views with incremental updates using Flink |1.0.0||
-|SQL DML support for Presto/Trino connectors (could be accelerated based on community feedback) |1.0.0||
-|Explore other execution engines/runtimes (Ray, native Rust, Python) |1.0.0||
+| Spark datasource V2 read | 0.13 | [HUDI-4449](https://issues.apache.org/jira/browse/HUDI-4449) |
+| Upgrade to Spark 3 as the default profile | 0.13 | [HUDI-3431](https://issues.apache.org/jira/browse/HUDI-3431) |
 
 ## Platform Services
 
 |Feature|Target Release|Tracking|
 |------------|--------|-----------|
-|Native support for AWS Glue Metastore   |0.11|[HUDI-2757](https://issues.apache.org/jira/browse/HUDI-2757)|
-|BigQuery and Snowflake external table integration   |0.12|[RFC-34](https://github.com/apache/hudi/pull/4503)|
-|JDBC Incremental Source GA   |0.12|[HUDI-1859](https://issues.apache.org/jira/browse/HUDI-1859)|
-|Mutable, CDC Stream support for Kafka Connect Sink   |0.12|[HUDI-2324](https://issues.apache.org/jira/browse/HUDI-2324)|
-|Airbyte integration   |0.12|RFC coming soon|
-|Apache Pulsar integration for Delta Streamer (blocked on upstream)   |0.12|[HUDI-246](https://issues.apache.org/jira/browse/HUDI-246)|
-|Kinesis deltastreamer source, with DynamoDB CDC   |0.12|[HUDI-1386](https://issues.apache.org/jira/browse/HUDI-1386), [HUDI-310](https://issues.apache.org/jira/browse/HUDI-310)|
-|Support for reliable, event based ingestion from cloud stores - GCS, Azure and the others   |1.0.0|[HUDI-1896](https://issues.apache.org/jira/browse/HUDI-1896)|
-|Hudi Timeline Metaserver for locks, column status and table listings (could be accelerated based on community feedback)   |1.0.0|[Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_0) |
-|Mutable, Transactional caching for Hudi Tables   (could be accelerated based on community feedback) |1.0.0|[Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_5)|
```

Comment:
saw that you've removed the original 1.0.0 items. i think it's still important to keep and show some perspective into the future. wdyt @prasannarajaperumal @vinothchandar ?

Reply:
I can bring the original 1.0.0 items back.  I wasn't sure if they're committed but we can show the perspective as you said.